static/app/views/eventsV2/homepage.tsx

```diff
@@ -43,6 +44,9 @@ class HomepageQueryAPI extends AsyncComponent<Props, HomepageQueryState> {
         `/organizations/${organization.slug}/discover/homepage/`,
       ]);
     }
+    if (location.search === '') {
```

Comment:
Tbh we should probably leave a `// HACK` comment here explaining why we're controlling a key 😅